common/src/main/java/io/netty/util/internal/NativeLibraryLoader.java

```diff
@@ -96,10 +96,11 @@ public static void loadFirstAvailable(ClassLoader loader, String... names) {
                 load(name, loader);
                 return;
             } catch (Throwable t) {
+                logger.debug("Unable to load the library '{}', trying next name...", name);
```

Comment:
instead of logging each time we fail, can we just log a single summary line that summarizes:
- if we failed, where are the places we attempted to load from
- if we succeeded, where did it actually load from

the `Unable to load the library` confuses folks to think something is wrong when it often isn't bcz the library is eventually loaded.

Reply:
I still thought it may be useful... what about keep these but log in trace ?

Reply:
The individual log lines often confuses users and results in support questions (e.g. why isn't my library loading, this is scary). The visibility at DEBUG is nice to understand what is/isn't loaded and I think if we can just have a single log line (either it worked or it didn't) would resolve the confusion.